### PR TITLE
Sql Added job points as per https://github.com/Windower/Resources/blob/master/lua/job_points.lua

### PR DESCRIPTION
**Pull request by some1here**
https://github.com/Windower/Resources/blob/master/lua/job_points.lua

### CODE REVIEW

File: sql/job_points.sql [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+INSERT INTO `job_points` VALUES ('460', 'divine_emblem_effect', '2', '7');
+INSERT INTO `job_points` VALUES ('462', 'sepulcher_duration', '1', '7');
+INSERT INTO `job_points` VALUES ('464', 'palisade_effect', '1', '7');
+INSERT INTO `job_points` VALUES ('466', 'enlight_effect', '1', '7');
+INSERT INTO `job_points` VALUES ('512', 'blood_weapon_effect', '2', '8');
+INSERT INTO `job_points` VALUES ('514', 'arcane_circle_effect', '1', '8');
+INSERT INTO `job_points` VALUES ('516', 'soul_enslavement_effect', '1', '8');
+INSERT INTO `job_points` VALUES ('518', 'last_resort_effect', '2', '8');
+INSERT INTO `job_points` VALUES ('520', 'souleater_duration', '1', '8');
+INSERT INTO `job_points` VALUES ('522', 'weapon_bash_effect, '1', '8');
```
Comment (bendangelo) [inline sql/job_points.sql]: Missing quote.

**Comment by Topace1** [comment]
This means job points will be obtainable in game?

**Comment by some1here** [comment]
This is part of it.   darkstar/src/map/job_points.cpp has to be done and the Nomad Moogle in Ru'Lude Gardens needs his update to start the quest

**Comment by teschnei** [comment]
Heh. It's the first step of like 50 to make job points work

**Comment by bendangelo** [comment]
Ok I don't see any problem merging this. Any objections?

**Comment by teschnei** [comment]
make sure the item_basic changes are legit - the job points sql file is fine

**Comment by xdemolish** [comment]
closing and reopening on the same branch doesnt do anything besides spam everyone and is only gonna  delay someone reviewing

also test before committing, the reason its not passing our checks is because you didnt test

**Comment by some1here** [comment]
It let me add another file changed WS

**Comment by some1here** [comment]
I also wonder if the Info has been tested from WINDOWER SE scans
https://github.com/Windower/Resources/tree/master/lua

**Comment by TeoTwawki** [comment]
> It let me add another file changed WS

You can add/change/remove without closing, as you have been told over and over many times. And your files are still broken. https://github.com/DarkstarProject/darkstar/pull/2949/files#diff-245f26bee42081c15928ebf025d06b68R27

**Comment by TeoTwawki** [comment]
If you test locally you'll see these things before you waste all that time pushing a dozen new commits that don't fix it:

[Importing item_basic.sql into the database...ERROR 1064 (42000) at line 16639: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'haruspex_slops',1,2080,0,1,0); at line 1](https://travis-ci.org/DarkstarProject/darkstar/builds/120477617))

**Comment by bendangelo** [comment]
Tests are failing, please fix. Also someone has to check the item changes. And do not add more stuff to this PR please. It will only keep it open longer. Open a new PR for more changes. Thanks.

**Comment by some1here** [comment]
It errors the ghost of item 16398's past........ https://github.com/DarkstarProject/darkstar/pull/2949/commits/cb6c1dfcdad4bdff3206ebe902425b2f8cc2b1b9
Removal can be taken back ....... but this shows how messed up it is

**Comment by xdemolish** [comment]
you still haven't tested
we won't merge if you won't test

File: sql/item_basic.sql [hunk 2]
```diff
@@ -646,16 +645,16 @@ INSERT INTO `item_basic` VALUES (729,0,'bloodwood_log','bloodwood_log',12,0,43,0
 INSERT INTO `item_basic` VALUES (730,0,'piece_of_bloodwood_lumber','bldwd_lumber',12,0,43,0,1387);
 INSERT INTO `item_basic` VALUES (731,0,'aquilaria_log','aquilaria_log',12,0,43,0,0);
 INSERT INTO `item_basic` VALUES (732,0,'kapor_log','kapor_log',12,0,43,0,0);
-INSERT INTO `item_basic` VALUES (733,0,'piece_of_kapor_lumber','kapor_lumber',12,0,0,1,0);
-INSERT INTO `item_basic` VALUES (734,0,'chunk_of_thokcha_ore','thokcha_ore',12,0,0,1,0);
-INSERT INTO `item_basic` VALUES (735,0,'thokcha_ingot','thokcha_ingot',12,0,0,1,0);
```
Comment (bendangelo) [inline sql/item_basic.sql]: Removed a zero here.

**Comment by bendangelo** [comment]
If you open a new PR with just the job points changes I'll merge that in. At the moment I don't have time to check all the item changes.

File: sql/item_basic.sql (hunk 2 above)
Comment (some1here) [inline sql/item_basic.sql]: That allows Sale on AH

Comment (some1here) [inline sql/item_basic.sql]: take for instance INSERT INTO `item_basic` VALUES (12467,0,'wool_cap','wool_cap',1,2080,17,0,4234);

Comment (some1here) [inline sql/item_basic.sql]: CREATE TABLE `item_basic` (
  `itemid` smallint(5) unsigned NOT NULL,
  `subid` smallint(4) unsigned NOT NULL DEFAULT '0',
  `name` tinytext NOT NULL,
  `sortname` tinytext NOT NULL,
  `stackSize` tinyint(2) unsigned NOT NULL DEFAULT '1',
  `flags` smallint(5) unsigned NOT NULL DEFAULT '0',
  `aH` tinyint(2) unsigned NOT NULL DEFAULT '99',
  `NoSale` tinyint(1) unsigned NOT NULL DEFAULT '0',
  `BaseSell` int(10) unsigned NOT NULL DEFAULT '0',

**Comment by some1here** [comment]
ok closing ON REQUEST. I will open another

**Comment by some1here** [comment]
https://github.com/DarkstarProject/darkstar/pull/3014

**Pull request closed without merging**